src/components/adjudications/AdjudicationDialog.jsx

```diff
@@ -68,17 +67,16 @@ class AdjudicationDialog extends React.Component {
           <DialogReadOnly fullWidth label="Notes" defaultValue={notes} />
           {audio && <AudioPlay fileName="file_test.mp3" time="3:07" />}
           <FormHelperText>Award Considerations</FormHelperText>
-          {specialAward
-            && (
-              <div>
-                <LensIcon fontSize="inherit" color="inherit" style={{ color: 'purple' }} />
-                Special Award
-              </div>
-            )}
+          {specialAward && (
+            <div>
+              <LensIcon fontSize="inherit" color="primary" />
+              Special Award
+            </div>
+          )}
           {choreoAward
```

Comment:
Like the change above, let's bring this && up a line to be with the `choreoAward`